x/evm/keeper/chainKeeper.go

```diff
@@ -717,8 +718,28 @@ func (k chainKeeper) setUnsignedCommandBatchID(ctx sdk.Context, id []byte) {
 }
 
 // returns the queue of commands
-func (k chainKeeper) getCommandQueue(ctx sdk.Context) utils.BlockHeightKVQueue {
-	return utils.NewBlockHeightKVQueue(commandQueueName, k.getStore(ctx, k.chainLowerKey), ctx.BlockHeight(), k.Logger(ctx))
+func (k chainKeeper) getCommandQueue(ctx sdk.Context) utils.GeneralKVQueue {
+	return utils.NewGeneralKVQueue(
+		commandQueueName,
+		k.getStore(ctx, k.chainLowerKey),
+		k.Logger(ctx),
+		func(value codec.ProtoMarshaler) utils.Key {
+			command, ok := value.(*types.Command)
+			if !ok {
+				panic(fmt.Errorf("unexpected type of command %T", command))
+			}
+
+			bz := make([]byte, 8)
+
+			switch command.Command {
+			case types.AxelarGatewayCommandBurnToken:
```

Comment:
why is there no prefix at all in this case, shouldn't it be 0 or something?

Reply:
`0` is just empty bytes and by doing nothing to `bz`, it is. @cgorenflo